inc/vcf/file_structure.hpp

```diff
@@ -168,88 +168,185 @@ namespace ebi
         /**
          * Checks that chromosome does not contain colons or white-spaces
          * 
-         * @throw std::invalid_argument
+         * @throw ChromosomeBodyError
          */
         void check_chromosome() const;
-        
+
         /**
-         * Checks that IDs are alphanumeric
+         * Checks that chromosome does not contain any colons
          * 
-         * @throw std::invalid_argument
+         * @throw ChromosomeBodyError
+         */
+        void check_chromosome_no_colons() const;
+
+        /**
+         * Checks that chromosome does not contain any white-spaces
+         * 
+         * @throw ChromosomeBodyError
+         */
+        void check_chromosome_no_whitespaces() const;
+
+        /**
+         * Checks that IDs are alphanumeric and do not contain duplicate values
+         * 
+         * @throw IdBodyError
          */
         void check_ids() const;
         
         /**
+         * Checks that ID contains no semicolons or white-spaces
+         * 
+         * @throw IdBodyError
+         */
+        void check_ids_no_semicolons_whitespaces() const;
+
+        /**
+         * Checks that ID contains no duplicate values in the same line
+         * 
+         * @throw IdBodyError
+         */
+        void check_ids_no_duplicates() const;
+
+        /**
          * Checks the structure of an alternate allele and its accordance to the meta section
          * 
-         * @throw std::invalid_argument
+         * @throw AlternateAllelesBodyError
          */
         void check_alternate_alleles() const;
         
         /**
-         * Checks the structure of an alternate allele:
+         * Checks the structure of an alternate allele against the reference:
          * - The dot is not combined with others
          * - Is not the same as the reference
          * - Shares the first nucleotide with the reference (does not apply to SV, break-ends and custom ALTs)
          * 
-         * @throw std::invalid_argument
-         */
+         * @throw AlternateAllelesBodyError
+	 */
         void check_alternate_allele_structure(std::string const & alternate, RecordType type) const;
         
         /**
+         * Check that alternates of the form <SOME_ALT> begin with DEL, INS, DUP, INV or CNV
+         * 
+         * @throw AlternateAllelesBodyError
+         */
+        void check_alternate_allele_symbolic_prefix(std::string const & alternate) const;
+
+        /**
          * Checks that alternates of the form <SOME_ALT_ID> are described in the meta section
          * 
-         * @throw std::invalid_argument
+         * @throw AlternateAllelesBodyError
          */
         void check_alternate_allele_meta(std::string const & alt_id,
```

Comment:
This method is implemented in validate_optional_policy, and not implemented or used as record method, so I would remove this prototype